assets/scss/base.scss

```diff
@@ -184,3 +184,157 @@ svg.svg-inline--fa{
 .tinymce-textarea {
   background-color: $backgroundWhite;
 }
+
+
+body {
+  font-size: .875rem;
+}
+
+.feather {
+  width: 16px;
+  height: 16px;
+  vertical-align: text-bottom;
+}
+
+/*
+ * Sidebar
+ */
+
+.sidebar {
+  position: fixed;
+  top: 0;
+
+  /* rtl:raw:
+  right: 0;
+  */
+  bottom: 0;
+
+  /* rtl:remove */
+  left: 0;
+  z-index: 100;
+
+  /* Behind the navbar */
+  padding: 48px 0 0;
+
+  /* Height of navbar */
+  box-shadow: inset -1px 0 0 rgba(0, 0, 0, 0.1);
+}
+
+@media (max-width: 767.98px) {
+  .sidebar {
+    top: 2rem;
+  }
+}
+
+.sidebar-sticky {
+  position: relative;
+  top: 0;
+  height: calc(100vh - 48px);
+  padding-top: .5rem;
+  overflow-x: hidden;
+  overflow-y: auto;
+
+  /* Scrollable contents if viewport is shorter than content. */
+}
+
+.sidebar .nav-link {
+  font-weight: 500;
+  color: #333;
+
+  .feather {
+    margin-right: 4px;
+    color: #727272;
+  }
+
+  &.active {
+    color: #2470dc;
+  }
+
+  &:hover .feather, &.active .feather {
+    color: inherit;
+  }
+}
+
+.sidebar-heading {
+  font-size: .75rem;
+  text-transform: uppercase;
+}
+
+/*
+ * Navbar
+ */
+
+.navbar-brand {
+  padding-top: .75rem;
+  padding-bottom: .75rem;
+  font-size: 1rem;
+  background-color: rgba(0, 0, 0, 0.25);
+  box-shadow: inset -1px 0 0 rgba(0, 0, 0, 0.25);
```

Comment:
A mixin would be great here so that you can use the same shadow in other places as well.